pages/linux/halt.md

```diff
@@ -1,16 +1,24 @@
 # halt
 
-> Halt, power-off or reboot the machine.
-> More information: <https://www.freedesktop.org/software/systemd/man/halt.html>.
+> Halt the system.
+> More information: <https://man7.org/linux/man-pages/man8/reboot.8.html>.
```

Comment:
Why reboot here?
```suggestion
> More information: <https://www.man7.org/linux/man-pages/man8/halt.8.html>.
```

Reply:
Because that url covers all three commands and I didn't know there was a url for reboot too